src/Compression/CompressionCodecQuantile.cpp

```diff
@@ -0,0 +1,218 @@
+#include "config.h"
+
+#ifdef USE_PCO
+#if USE_PCO
+
+#include "base/types.h"
+#include <Common/Exception.h>
+#include <Compression/ICompressionCodec.h>
+#include <Compression/CompressionFactory.h>
+#include <Compression/CompressionInfo.h>
+#include <DataTypes/IDataType.h>
+#include <Parsers/ASTLiteral.h>
+
+#include <pco.h>
+
+namespace DB
+{
+
+class CompressionCodecQuantile : public ICompressionCodec
```

Comment:
Why is this referred to as `Quantile`? Quantile Compression was the predecessor to Pcodec, and I think we should be clear that we're using Pcodec here.